_docs/tasks/traffic-management/ingress.md

```diff
@@ -170,7 +170,7 @@ to configure ingress behavior.
    
    Notice that in this example we are only exposing httpbin's `/ip` endpoint.
    
-   > Note: Envoy currently only allows a single TLS secret in the ingress since SNI is not yet supported.
+   > Note: Envoy currently only allows a single TLS secret in the ingress since SNI is not yet supported. That means that the secret name field in ingress resource is not used, and the secret must be called `istio-ingress-certs` in `istio-ingress` namespace.
```

Comment:
you mean in istio-system namespace

Reply:
this was intentional to mean the namespace, in which istio-ingress resides, but I changed it per your suggestion.